app/models/ability.rb

```diff
@@ -10,12 +10,19 @@ class Ability
   def custom_permissions
     # Limits deleting objects to a the admin user
     #
-    can [:manage], :all if admin?
+    can [:manage], :all if current_user.has_role? :admin
```

Comment:
Oh, is that private method below only to satisfy rolify? I saw the duplication in code and balked.